README.md

```diff
@@ -1 +1,33 @@
 # Proyecto Asistente Virtual para Cursos 2024
+
+
+# prueba de registro
+# user: admin, password: admin123
```

Comment:
Esto no corresponde porque al volver a ejecutar la base de datos esto no aplica. Capaz podríamos poner una sección que sea primero debes registrarte, luego ingresar desde la pantalla Log In. El resto del flujo, posterior al Log In, no es necesario porque lo vamos a cambiar seguramente